src/main/java/com/paneedah/weaponlib/WeaponSpawnEntity.java

```diff
@@ -138,11 +139,14 @@ protected void onImpact(RayTraceResult position) {
             //Projectiles projectilesConfig = weapon.getModContext().getConfigurationManager().getProjectiles();
 
             if(this.getThrower() != null) {
-                position.entityHit.attackEntityFrom(DamageSource.causeThrownDamage(this, this.getThrower()), damage);
+				position.entityHit.attackEntityFrom(DamageSource.causeThrownDamage(this, this.getThrower()), damage);
             } else {
                 position.entityHit.attackEntityFrom(new DamageSource("thrown"), damage);
             }
 
+			if(!ModernConfigManager.knockbackOnHit)
+				position.entityHit.setVelocity(this.motionX / 1500, this.motionY / 1500, this.motionZ / 1500);
```

Comment:
Still please fix the spacing

Reply:
Which line

Reply:
142, 147, 148.
Are you not seeing it?
![image](https://github.com/Cubed-Development/Modern-Warfare-Cubed/assets/82710983/d9f18cd6-acb0-464e-ad3a-f92ad74c9c3c)

Reply:
![image](https://github.com/Cubed-Development/Modern-Warfare-Cubed/assets/123817318/c9109bd4-4096-4ba2-ad13-855dae213367)

Reply:
There's no problem with this

Reply:
wat is that...
Well let's just hope it's just a git bug and everything will be okay.